docs/ReactQt/DevEnvSetup.md

```diff
@@ -0,0 +1,60 @@
+# Prerequisites:
+lein, node.js v.8 , cmake, Qt 5.9.1 (with QtWebEngine components installed), Qt's qmake available in PATH
+
+Note: add qmake to PATH via
+`export PATH=<QT_PATH>/clang_64/bin:$PATH`
+
+Caveats:
+  - if npm hangs at some step, check the version. If it's 5.6.0, try downgrading to 5.5.1 via `npm install -g npm@5.5.1`
+
+# To install react-native-cli with desktop commands support:
+1. git clone https://github.com/status-im/react-native-desktop.git
+2. cd react-native-desktop/react-native-cli
+3. npm update
+4. npm install -g
+
+# To setup re-natal dev builds of status-react for Desktop:
```

Comment:
@siphiuel, thank you for putting all notes together!
What do you think about splitting them a bit? 

`react-native-desktop` project for our team is tightly related with `status-react`. But for another developer who looks for porting his react-native app to desktop, all mentions of `status-react` will be unknown and confusing. Also, other developers can use `react-native-desktop` without clojurescript.

I think that explanations of how to setup `status-react` dev builds with figwheel, how to fix go errors or deal with realm, etc... should go to `status-react` repository documentation.
I believe that separate doc can be arranged for instructions on how to create react-native-desktop apps with clojurescript. Troubleshooting for `react-native-desktop` and for `status-react` for desktop can become two separate documents.

What do you think?